charts/linkerd/values.yaml

```diff
@@ -0,0 +1,165 @@
+      NqCzU54MXTMUEaNQBNcHyzz8xfZ2vZY56TQQod3HQ8dEcPwRNgTFJxgzdt03o0Iw
+      QDAOBgNVHQ8BAf8EBAMCAQYwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMC
+      MA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwIDSAAwRQIhAJpWjlDgXGo8hgvr
+      WX/H+QmI4sSA3EiW1J9ZNweS94UsAiB9IN9rkv/b47tYsynZCpLTzSeF7+16VdyI
+      DT1weqC2Vg==
+      -----END CERTIFICATE-----
+
+    # Must match the expiry date in CrtPEM
+    CrtExpiry: 2020-07-25T03:17:08Z
+    CrtExpiryAnnotation: linkerd.io/identity-issuer-expiry
+
+    # PEM encode ECDSA private key
+    KeyPEM: |
+      -----BEGIN EC PRIVATE KEY-----
+      MHcCAQEEINfi1j8CjonK4FCuDoCUPXRxVMNResGsQDbFqEXTUeJUoAoGCCqGSM49
+      AwEHoUQDQgAE8DxSABAKLIqfVqaLxcNzKyUN1jags1OeDF0zFBGjUATXB8s8/MX2
+      dr2WOek0EKHdx0PHRHD8ETYExScYM3bdNw==
+      -----END EC PRIVATE KEY-----
+  Resources:
+    CPU:
+      <<: *controller_resources_cpu
+    Memory: *controller_resources_memory
+
+# grafana configuration
+GrafanaImage: gcr.io/linkerd-io/grafana
+GrafanaResources:
+  <<: *controller_resources
+
+# heartbeat configuration
+HeartbeatResources:
+  <<: *controller_resources
+HeartbeatSchedule: "0 0 * * * "
+
+# prometheus configuration
+PrometheusImage: prom/prometheus:v2.11.1
+PrometheusLogLevel: *controller_log_level
+PrometheusResources:
+  <<: *controller_resources
+
+# proxy configuration
+Proxy:
+  ClusterDomain: *cluster_domain
+  ControlPlaneNamespace: *namespace
+  DisableIdentity: false
+  EnableExternalProfile: false
+  HighAvailability: *high_availability
+  Identity:
+    TrustDomain: *cluster_domain
+    TrustAnchors: |
+      -----BEGIN CERTIFICATE-----
+      MIIBgzCCASmgAwIBAgIBATAKBggqhkjOPQQDAjApMScwJQYDVQQDEx5pZGVudGl0
+      eS5saW5rZXJkLmNsdXN0ZXIubG9jYWwwHhcNMTkwNzI2MDMxNjQ4WhcNMjAwNzI1
+      MDMxNzA4WjApMScwJQYDVQQDEx5pZGVudGl0eS5saW5rZXJkLmNsdXN0ZXIubG9j
+      YWwwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAATwPFIAEAosip9WpovFw3MrJQ3W
+      NqCzU54MXTMUEaNQBNcHyzz8xfZ2vZY56TQQod3HQ8dEcPwRNgTFJxgzdt03o0Iw
+      QDAOBgNVHQ8BAf8EBAMCAQYwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMC
+      MA8GA1UdEwEB/wQFMAMBAf8wCgYIKoZIzj0EAwIDSAAwRQIhAJpWjlDgXGo8hgvr
+      WX/H+QmI4sSA3EiW1J9ZNweS94UsAiB9IN9rkv/b47tYsynZCpLTzSeF7+16VdyI
+      DT1weqC2Vg==
+      -----END CERTIFICATE-----
+  Image:
+    Name: gcr.io/linkerd-io/proxy
+    PullPolicy: IfNotPresent
+  LogLevel: warn,linkerd2_proxy=info
+  Port: &proxy_ports
+    Admin: 4191
+    Control: 4190
+    Inbound: 4143
+    Outbound: 4140
+  UID: &proxy_uid 2102
+
+  # use this to override the default resource requirements in HA mode
+  ResourceRequirements:
+    CPU:
+      Limit: "1"
+      Request: 100m
+    Memory:
+      Limit: 250Mi
+      Request: 20Mi
+
+# proxy-init configuration
+ProxyInit:
+  Image:
+    Name: gcr.io/linkerd-io/proxy-init
+    PullPolicy: IfNotPresent
+    Version: v1.0.0
+  Proxy:
+    Port:
+      <<: *proxy_ports
+      IgnoreInboundPorts: ""
+      IgnoreOutboundPorts: "443"
+    UID: *proxy_uid
```

Comment:
For the main `Proxy` section and this `ProxyInit` section to share the same underlying struct, it might make sense to put these `Port` and `UID` sections as direct descendants of `ProxyInit`.